bpf/lib/lb.h

```diff
@@ -632,26 +634,33 @@ lb6_select_backend_id(struct __ctx_buff *ctx __maybe_unused,
 		      const struct ipv6_ct_tuple *tuple,
 		      const struct lb6_service *svc)
 {
-	__u32 zero = 0, index = svc->rev_nat_index, lut_size = LB_MAGLEV_LUT_SIZE;
+	__u32 index = svc->rev_nat_index, slot = 0, offset = 0,
+	      lut_size = LB_MAGLEV_LUT_SIZE;
 	__u16 *backend_ids;
 	void *maglev_lut;
 
 	maglev_lut = map_lookup_elem(&LB6_MAGLEV_MAP_OUTER, &index);
 	if (unlikely(!maglev_lut))
 		return 0;
 
-	backend_ids = map_lookup_elem(maglev_lut, &zero);
+	backend_ids = map_lookup_elem(maglev_lut, &slot);
 	if (unlikely(!backend_ids))
```

Comment:
Why is the lookup in this commit still needed? The backend_ids are not used anywhere and later overridden, no?